### ISSUE TITLE
Mod isnt working on 1.20.1 Forge

### ISSUE THREAD

**Issue by spunchbop121** [FusionSwarly/Create-Structures#2]
advancementframes-1.20-2.0.2.jar
architectury-9.1.10-forge.jar
carpeted-1.20-1.4.jar
cloth-config-11.0.99-forge.jar
create-1.20.1-0.5.1.d.jar
create-structures-0.1.1-1.20.1-FORGE.jar
dummmmmmy-1.20-1.7.9.jar
emi-1.0.11+1.20.1+forge.jar
Etcetera-1.20.1-1.0.2-Forge.jar
fastpaintings-1.20-1.1.1.jar
goated-1.20-1.3.3.jar
hauntedharvest-1.20-3.1.10.jar
heartstone-1.20-1.1.3.jar
labels-1.20-1.13.jar
lootr-1.20-0.7.30.73.jar
modelfix-1.14.jar
modernfix-forge-5.3.1+mc1.20.1.jar
moonlight-1.20-2.7.0-forge.jar
MouseTweaks-forge-mc1.20-2.25.jar
moyai-1.20-2.0.2.jar
mysticaloaktree-1.20-1.11.jar
randomium-1.20-1.28.jar
sleep_tight-1.20-1.1.4.jar
smarterfarmers-1.20-1.7.3.jar
snowyspirit-1.20-2.2.1.jar
sootychimneys-1.20.1-1.1.1.jar
Species-1.20.1-1.3-Forge.jar
stepitup-2.0-1.20.1-forge.jar
supplementaries-1.20-2.5.21.jar
suppsquared-1.20-1.1.6.jar
twigs-1.20.1-3.1.0-forge.jar

**Comment by ffuentesm** [comment]
Yeah, I have to say that I have been playing on 1.20 for a while now, and I have not encountered a single structure from this mod.

**Comment by MichaelMichelle2** [comment]
Yes I can agree, this is not working unless you use the datapack version of it.

**Comment by dhouck** [comment]
I donʼt know about Forge, but I seem to have the same problem on Fabric.  I *think* (but could be reading this wrong) that it is trying to load itself as a resource pack, not a data pack, which obviously wonʼt work right.
